samples/angular-todo/src/app/features/journey/webAuthn/webAuthn.component.html

```diff
@@ -0,0 +1,38 @@
+<!--
+angular-todo-prototype
+
+webAuthn.component.html
+
+Copyright (c) 2021 ForgeRock. All rights reserved.
+This software may be modified and distributed under the terms
+of the MIT license. See the LICENSE file for details.
+-->
+
+<ng-container
+  *ngTemplateOutlet="
+    success ? successMessage : failure ? failureMessage : loading;
+    context: { message: { message: message, header: header } }
+  "
```

Comment:
oh yes, that's basically a similar logic being applied at the form component.  It works as an IF statement, depending on the value of those variables 'success' and 'failure' it will load a template (which is below), either successMessage or failureMessage or loading template. At the same time, by passing the 'context' atribute, it sends an object to be used by the template.